include/swift/IDE/CodeCompletion.h

```diff
@@ -594,14 +594,15 @@ class CodeCompletionResult {
   unsigned SemanticContext : 3;
   unsigned NotRecommended : 1;
   unsigned NotRecReason : 3;
+  unsigned IsSystem : 1;
 
   /// The number of bytes to the left of the code completion point that
   /// should be erased first if this completion string is inserted in the
   /// editor buffer.
-  unsigned NumBytesToErase : 7;
+  unsigned NumBytesToErase : 6;
 
 public:
-  static const unsigned MaxNumBytesToErase = 127;
+  static const unsigned MaxNumBytesToErase = 63;
```

Comment:
I think 63 is still more than enough.

Reply:
@benlangmuir WDYT?

Reply:
As per offline discussion, we decided to keep it 127.